src/defaultConfig.ts

```diff
@@ -1820,11 +1822,12 @@ const config = {
   },
 };
 
-config.class.arrowMarkerAbsolute = config.arrowMarkerAbsolute;
-config.gitGraph.arrowMarkerAbsolute = config.arrowMarkerAbsolute;
+console.log(JSON.stringify(config));
+if (config.class) config.class.arrowMarkerAbsolute = config.arrowMarkerAbsolute;
+if (config.gitGraph) config.gitGraph.arrowMarkerAbsolute = config.arrowMarkerAbsolute;
```

Comment:
These are great! The less we mess with existing integrations the better. We should setup a deprecation waring system.